doc/devel/testing.rst

```diff
@@ -53,58 +56,72 @@ value.
 Running the tests
 -----------------
 
-Running the tests is simple. Make sure you have nose installed and run::
+Running the tests is simple. Make sure you have pytest installed and run::
+
+   py.test
+
+or::
 
    python tests.py
 
 in the root directory of the distribution. The script takes a set of
 commands, such as:
 
 ========================  ===========
-``--pep8``                pep8 checks
-``--no-pep8``             Do not perform pep8 checks
+``--pep8``                Perform pep8 checks (requires pytest-pep8_)
 ``--no-network``          Disable tests that require network access
 ========================  ===========
 
-Additional arguments are passed on to nosetests. See the nose
-documentation for supported arguments. Some of the more important ones are given
-here:
+Additional arguments are passed on to pytest. See the pytest documentation for
+supported arguments. Some of the more important ones are given here:
```

Comment:
Might be good to put a link to the pytest documents that list supported arguments